utils/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -75,18 +75,24 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
         }
     }
 
+    // In particular, NTLM (as used e.g. by Microsoft TFS) is not supported due to Java's limiting Authenticator
+    // mechanism.
```

Comment:
I don't understand the reasoning here.
Oracle writes specifically that NTLM is supported [1].

[1] https://docs.oracle.com/javase/8/docs/technotes/guides/net/http-auth.html

Reply:
Looks like I'm running into the part starting with "Because the Authenticator class was defined prior to NTLM being supported, it was not possible to add support in the API for the NTLM domain field" on that page. I simply cou'l get it to work with any of the mentioned methods.